crates/bevy_ui/src/texture_slice.rs

```diff
@@ -35,7 +36,7 @@ impl ComputedTextureSlices {
         &'a self,
         transform: &'a GlobalTransform,
         node: &'a Node,
-        background_color: &'a BackgroundColor,
+        color: LegacyColor,
```

Comment:
```suggestion
```
We can just use the `UiImage::color` field I think

Reply:
True, I didn't notice that.

Reply:
True, I missed this. Will update the PR.